packages/opentelemetry-tracing/src/BasicTracerProvider.ts

```diff
@@ -16,17 +16,18 @@
 
 import * as api from '@opentelemetry/api';
 import {
+  CompositePropagator,
   ConsoleLogger,
-  HttpTraceContext,
   HttpCorrelationContext,
-  CompositePropagator,
+  HttpTraceContext,
 } from '@opentelemetry/core';
+import { Resource } from '@opentelemetry/resources';
 import { SpanProcessor, Tracer } from '.';
-import { DEFAULT_CONFIG } from './config';
 import { MultiSpanProcessor } from './MultiSpanProcessor';
 import { NoopSpanProcessor } from './NoopSpanProcessor';
 import { SDKRegistrationConfig, TracerConfig } from './types';
-import { Resource } from '@opentelemetry/resources';
+import merge = require('lodash.merge');
```

Comment:
`require` returns an `any` type. We should at least assert the type. But doesn't `import * as merge from 'lodash.merge';` work?

Reply:
Sadly no

```
error TS2497: This module can only be referenced with ECMAScript imports/exports by turning on the 'esModuleInterop' flag and referencing its default export.

29 import * as merge from 'lodash.merge';
```

Reply:
not getting `any` though 
![image](https://user-images.githubusercontent.com/24660299/100954680-07528080-34e3-11eb-851d-716d495070d2.png)